mono/mini/mini-s390x.c

```diff
@@ -4677,10 +4677,11 @@ mono_arch_output_basic_block (MonoCompile *cfg, MonoBasicBlock *bb)
 		case OP_MEMORY_BARRIER:
 			s390_mem (code);
 			break;
-#if USE_COOP_GC
 		case OP_GC_SAFE_POINT: {
```

Comment:
No you can leave it as is, at least it's going to be here when we have safepoints on s390x, and it doesn't change anything in the meantime.